backend/tests/functional_tests/scripts/test_sync_configuration.py

```diff
@@ -373,6 +397,8 @@ def test_sync_with_changed_task_in_dataset(mocker: Any, prepare_environment: Any
     kidneys_segmentation = TasksRepository.get_task_by_key('KIDNEYS_SEGMENTATION')
     assert kidneys_segmentation.name == 'New Kidneys segmentation'
     assert kidneys_segmentation.image_path == 'assets/icon/kidneys_dataset_icon.svg'
+    assert kidneys_segmentation.description == 'This is a test task'
+    assert len(kidneys_segmentation.label_examples) == 2
```

Comment:
Can we also check what's inside of `kidneys_segmentation.label_examples`? At least in one place ;)